### ISSUE TITLE
Open Editing in Android Studio

### ISSUE THREAD

**Issue by mihir-chauhan** [flutter/flutter-intellij#4607]
## What happened

I clicked on Open Editing in Android Studio when in the android/app/build.gradle file. It gave this error when I clicked on the button.

## Version information

Android Studio `3.5.2` • Flutter plugin `io.flutter 44.0.1` • Dart plugin `191.8593`

Error getting Flutter sdk information.

## Exception

*** exception class was changed or removed

```
: com/android/tools/idea/gradle/project/ProjectImportUtil
	at io.flutter.actions.OpenAndroidModule.canImportAsGradleProject(OpenAndroidModule.java:100)
	at io.flutter.actions.OpenAndroidModule.openOrImportProject(OpenAndroidModule.java:60)
	at io.flutter.actions.OpenAndroidModule.actionPerformed(OpenAndroidModule.java:52)
	at io.flutter.actions.OpenInAndroidStudioAction.actionPerformed(OpenInAndroidStudioAction.java:49)
	at io.flutter.editor.NativeEditorNotificationProvider$NativeEditorActionsPanel.performAction(NativeEditorNotificationProvider.java:138)
	at com.intellij.ui.EditorNotificationPanel$1.hyperlinkActivated(EditorNotificationPanel.java:127)
	at com.intellij.ui.HyperlinkAdapter.hyperlinkUpdate(HyperlinkAdapter.java:28)
	at com.intellij.ui.HyperlinkLabel.fireHyperlinkEvent(HyperlinkLabel.java:222)
	at com.intellij.ui.HyperlinkLabel.processMouseEvent(HyperlinkLabel.java:145)
	at java.awt.Component.processEvent(Component.java:6315)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4899)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4532)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:766)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:739)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:736)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:757)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:702)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:375)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.android.tools.idea.gradle.project.ProjectImportUtil PluginClassLoader[io.flutter, 44.0.1] com.intellij.ide.plugins.cl.PluginClassLoader@47ed694
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:51)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 41 more
```

**Comment by devoncarew** [comment]
@mihir-chauhan, thanks for the issue. Android Studio 4.0 is now available, as is Flutter plugin v46.0. Can you try updating and see if that resolves the issue?

**Comment by mihir-chauhan** [comment]
@devoncarew Thanks for the answer. I updated Android Studio and now, I get this issue where all of my project imports and stuff is messed up. It says: "  error: failed linking references." when I try to build or compile the app.
